packages/render-fidelity-tools/test/config.json

```diff
@@ -358,6 +358,166 @@
         "dspbr-pt"
       ]
     },
+    {
+      "name": "khronos-ToyCar",
+      "model": "../../../shared-assets/models/glTF-Sample-Models/2.0/ToyCar/glTF-Binary/ToyCar.glb",
+      "orbit": {
+        "radius": 0.037,
+        "theta": -35,
+        "phi": 75
+      },
+      "dimensions": {
+        "width": 1200,
+        "height": 500
+      },
+      "target": {
+        "y": 0.001
+      },
+      "exclude": [
+        "dspbr-pt"
+      ]
+    },
+    {
+      "name": "khronos-IridescentDishWithOlives",
+      "model": "../../../shared-assets/models/glTF-Sample-Models/2.0/IridescentDishWithOlives/glTF-Binary/IridescentDishWithOlives.glb",
+      "orbit": {
+        "radius": 0.47
```

Comment:
Can we angle the camera so that we can see some of the olives through the glass? That will help with testing refraction. I'd also like to put a background on this one; maybe spruit sunrise? Just pick one of the backgrounds we're already using in another test.

Reply:
Can you also set a skybox? You can use MetalRoughSpheresHDR as an example.